nucypher/network/retrieval.py

```diff
@@ -140,7 +140,7 @@ def results(self) -> Tuple[List["RetrievalResult"], List[RetrievalError]]:
             results.append(
                 RetrievalResult(
                     {
-                        to_checksum_address(address): cfrag
+                        to_checksum_address(bytes(address)): cfrag
```

Comment:
This is a result of using rust's `Address`? These rust barriers are so explicit...

Reply:
Yes.  `to_checksum_address` does not accept the `Address` type.